spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -658,6 +660,58 @@ definitions:
           type: u8
           desc: Frequency slot. FCN+8 (that is [1..14]). 0 or 0xFF for invalid
 
+ - MSG_EPHEMERIS_GLO:
+    id: 0x0088
+    short_desc: Satellite broadcast ephemeris for GLO
+    desc: |
+      The ephemeris message returns a set of satellite orbit
+      parameters that is used to calculate GLO satellite position,
+      velocity, and clock offset. Please see the GLO ICD 5.1 "Table 4.5
+      Characteristics of words of immediate information (ephemeris parameters)"
+      for more details.
+    fields:
+      - common:
+          type: EphemerisCommonContent
+          desc: Values common for all ephemeris types
+      - gamma:
+          type: double
+          desc: Relative deviation of predicted carrier frequency from nominal
+      - tau:
+          type: double
+          desc: Correction to the SV time
+          units: s
+      - d_tau:
+          type: double
+          desc: Equipment delay between L1 and L2
+          units: s
+      - pos:
+          type: array
+          size: 3
+          fill: double
+          map_by: sid
+          desc: Position of the SV at tb in PZ-90.02 coordinates system
+          units: m
+      - vel:
+          type: array
+          size: 3
+          fill: double
+          map_by: sid
+          desc: Velocity vector of the SV at tb in PZ-90.02 coordinates system
+          units: m/s
+      - acc:
+          type: array
+          size: 3
+          fill: double
+          map_by: sid
+          desc: Acceleration vector of the SV at tb in PZ-90.02 coordinates sys
+          units: m/s^2
+      - fcn:
+          type: u8
+          desc: Frequency slot. FCN+8 (that is [1..14]). 0 or 0xFF for invalid
+      - iod:
+          type: u8
+          desc: Issue of ephemeris data
```

Comment:
This should have a note that this is the lower 7 bits of tb as defined in the RTCM 3.1 specification, for DF110